sdk/tables/Azure.Data.Tables/tests/TableServiceClientLiveTests.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.Testing;
+using Azure.Data.Tables.Models;
+using NUnit.Framework;
+
+namespace Azure.Data.Tables.Tests
+{
+    /// <summary>
+    /// The suite of tests for the <see cref="TableServiceClient"/> class.
+    /// </summary>
+    /// <remarks>
+    /// These tests have a dependency on live Azure services and may incur costs for the associated
+    /// Azure subscription.
+    /// </remarks>
+    //[LiveOnly]
+    public class TableServiceClientLiveTests : RecordedTestBase<TablesTestEnvironment>
+    {
+        /// <summary>
+        /// The table endpoint.
+        /// </summary>
+        private readonly Uri _tableEndpoint;
+
+        public TableServiceClientLiveTests(bool isAsync) : base(isAsync /* To record tests, add this argument, RecordedTestMode.Record */)
+        {
+            _tableEndpoint = new Uri(TestEnvironment.StorageUri);
```

Comment:
This is why tests are failing, recording is not loaded at this point yet. Inline this into `CreateTableServiceClient` and things should start working.

Reply:
I need to figure out how to make this more obvious...